pkg/collector/providers/file.go

```diff
@@ -295,5 +296,10 @@ func GetCheckConfigFromFile(name, fpath string) (check.Config, error) {
 		config.LogsConfig = rawLogsConfig
 	}
 
+	// DockerImages entry was found: we ignore it if no ADIdentifiers has been found
+	if cf.DockerImages != nil && cf.ADIdentifiers == nil {
```

Comment:
Although `[]string{nil}` is a thing, using that might break if the yaml lib changes it's behaviour to serialise in empty `[]string{}`s instead. WDYT about checking the length instead?